src/cldr/load.ts

```diff
@@ -1,14 +1,26 @@
 // required for Globalize/Cldr to properly resolve locales in the browser.
 import 'cldrjs/dist/cldr/unresolved';
-import { Require } from '@dojo/interfaces/loader';
-import Map from '@dojo/shim/Map';
 import load from '@dojo/core/load';
-import coreRequest, { Response } from '@dojo/core/request';
+import coreRequest from '@dojo/core/request';
+import { Provider } from '@dojo/core/request/interfaces';
 import has from '@dojo/has/has';
+import { Require } from '@dojo/interfaces/loader';
+import Map from '@dojo/shim/Map';
 import Promise from '@dojo/shim/Promise';
 import * as Globalize from 'globalize';
-import supportedMain from './locales';
 import { generateLocales } from '../util/main';
+import supportedMain from './locales';
+
+let requestProvider: Provider;
+
+if (!has('host-browser')) {
+	requestProvider = (<any> require('@dojo/core/request/providers/node')).default;
```

Comment:
Maybe not. If this can be removed, great -I can add this to the setup methods of the tests instead.  If we add this to the app, we're saying that anyone who uses widget-core is going to need, somewhere, a place where they set the default provider, regardless of whether they are directly using `request` or not.  If we're OK with that (or we can think of a better place to set the default), let's do it!